sphinx_autodoc_typehints.py

```diff
@@ -248,35 +246,31 @@ def _one_child(module):
     try:
         obj_ast = ast.parse(textwrap.dedent(inspect.getsource(obj)), **parse_kwargs)
     except TypeError:
-        return rv
+        return {}
 
     obj_ast = _one_child(obj_ast)
     if obj_ast is None:
-        return rv
+        return {}
```

Comment:
made these to make the code more readable, this way it's more explicit that we bail in these cases